docker-compose.yml

```diff
@@ -37,8 +37,8 @@ services:
       # - ./docker/db/data:/var/lib/mysql:rw
   nginx:
     build:
-      context: ./docker/nginx
-      dockerfile: ./Dockerfile
+      context: ./
+      dockerfile: ./Dockerfile.nginx
```

Comment:
If we switch to this convention, shouldn't we do it for all Dockerfiles (for consistency)?